cpu/lpc2387/cpu.c

```diff
@@ -166,6 +166,11 @@ bool cpu_woke_from_backup(void)
         return false;
     }
 
+    /* When we wake from Deep Sleep only POR is set, just like in the real
+     * POR case. Clear the bit to create a new, distinct state.
+     */
+    RSIR |= RSIR_POR;
```

Comment:
This doesn't clear the bit

Reply:
Yes it does.
![RSIR](https://user-images.githubusercontent.com/1301112/70437536-1c413300-1a8c-11ea-87a0-7833de98c552.png)

Reply:
This whole "clear on writing a one" style is super strange to me, but apparently quite popular.